.build/config.toml

```diff
@@ -0,0 +1,88 @@
+########################################
+#                                      #
+#       rust-rxpy configuration        #
+#                                      #
+########################################
+###################################
+#         Global settings         #
+###################################
+# Both or either one of http/https ports must be specified
+listen_port = 80
+listen_port_tls = 443
+

```

Comment:
Here, it is better to include the following lines for the latest update.

```toml
# Optional. If you listen on a custom port like 8443 but redirect with firewall to 443
# When you specify this, the server sends a redirection response 301 with specified port to the client for plaintext http request.
# Otherwise, the server sends 301 with the same port as `listen_port_tls`.
# https_redirection_port = 443
```

Reply:
I've just added it.

Reply:
Thanks!